src/com/aca/homework/week4/snake/Board.java

```diff
@@ -0,0 +1,105 @@
+package com.aca.homework.week4.snake;
+
+
+import java.util.Random;
+import java.util.Scanner;
+
+public class Board {
+
+    public static void main(String[] args) {
+        Snake[][] board = new Snake[8][8];
+
+        for (int i = 0; i < 8; i++) {
```

Comment:
to write a cleaner code I suggest moving this loop to a separate method with a name prepareInitialBoard